v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/utils/KafkaCommonUtils.java

```diff
@@ -162,4 +164,21 @@ public static Map<String, Object> configureKafka(Map<String, String> props) {
     }
     return config;
   }
+
+  public static Map<String, Object> configureKafkaOffsetCommit(KafkaReadOptions options) {
+    Map<String, Object> kafkaConfig = new HashMap<>();
+    if (options.getEnableCommitOffSets() && options.getConsumerGroupID().isBlank()) {
```

Comment:
I think `options.getConsumerGroupID()` can be null? If so, `.isBlank()` can cause a NPE.

Reply:
I provide a default value - `""`, which is a blank value right?

Reply:
Yeah, this should work. Up to you if you want to add a stronger check here.